src/__tests__/fixtures/users.ts

```
@@ -0,0 +1,134 @@
+export const testUsers = {
+  defaultUser: {
+    id: 'test-user-id',
+    aud: 'authenticated',
+    role: 'authenticated',
+    email: 'test@example.com',
+    email_confirmed_at: '2024-01-01T00:00:00.000Z',
+    phone: null,
+    phone_confirmed_at: null,
+    confirmed_at: '2024-01-01T00:00:00.000Z',
+    last_sign_in_at: '2024-01-01T00:00:00.000Z',
+    app_metadata: { 
```

ESLint

[error] Delete `·`

(prettier/prettier)